include/blmc_robots/n_joint_blmc_robot_driver.hpp

```diff
@@ -307,7 +315,12 @@ struct NJointBlmcRobotDriver<Observation, N_JOINTS, N_MOTOR_BOARDS>::Config
         Vector kd = Vector::Zero();
     } position_control_gains;
 
-    //! \brief Offset between home position and zero.
+    //! @brief Lower limits for joint position.
+    Vector joint_lower_limits = Vector::Zero();
+    //! @brief Upper limits for joint position.
+    Vector joint_upper_limits = Vector::Zero();
```

Comment:
this is a detail, but maybe it makes sense to make sure the default the set of admissible joint angles is empty (and does not contain 0, as it does now). e.g. by setting lower limits to Ones or sth like that

Reply:
I don't think that this would be a problem in practice. The only valid position would be all joints exactly zero which would immediately be violated on the real system if only due to slight vibration.
These are also not really default values that are used anywhere as the values need to be specified explicitly in the configuration file (otherwise loading fails).